src/projection/node/create-projection-node.ts

```diff
@@ -566,8 +564,19 @@ export function createProjectionNode<I>({
             sync.preRender(this.updateProjection, false, true)
         }
 
-        scheduleUpdateFailedCheck() {
-            sync.postRender(this.checkUpdateFailed)
+        scheduleCheckAfterUnmount(nodeInLayoutGroup?: IProjectionNode) {
+            /**
+             * If the unmounting node is in a layoutGroup and did trigger a willUpdate,
+             * we manually call didUpdate to give a chance to the siblings to animate.
+             * Otherwise, cleanup all snapshots to prevents future nodes from reusing them.
+             */
+            sync.postRender(() => {
+                if (nodeInLayoutGroup?.isLayoutDirty) {
```

Comment:
Currently this fires on `root` but that was only so multiple elements could schedule just one callback that removed all snapshots.

If you're doing it this way I think you can just call `projection.schedueCheckAfterUnmount`

Reply:
I didn't quite understand, don't we want to same for `didUpdate` if multiple elements schedule it?

Reply:
That's a good point but in this case didUpdate could be fired by an unlimited number of elements